fhd_utils/l_m_n.pro

```diff
@@ -0,0 +1,26 @@
+pro l_m_n, obs, psf, obsdec=obsdec, obsra=obsra, dec_arr=dec_arr, ra_arr=ra_arr,$
+    l_mode=l_mode,m_mode=m_mode,n_tracked=n_tracked
+
+if ~keyword_set(obsdec) then obsdec = obs.obsdec
+if ~keyword_set(obsra) then obsra = obs.obsra
+if ~keyword_set(dec_arr) then dec_arr = (*psf.image_info).dec_arr
+if ~keyword_set(ra_arr) then ra_arr = (*psf.image_info).ra_arr
+
+;Calculate l mode, m mode, and phase-tracked n mode of pixel centers
+cdec0 = cos(obsdec*!dtor)
+sdec0 = sin(obsra*!dtor)
+cdec = cos(dec_arr*!dtor)
+sdec = sin(dec_arr*!dtor)
+cdra = cos((ra_arr-obsra)*!dtor)
+sdra = sin((ra_arr-obsra)*!dtor)
+l_mode = cdec*sdra
+m_mode = sdec*cdec0 - cdec*sdec0*cdra
+;n=1 at phase center, so reference from there for phase tracking
+n_tracked = (sdec*sdec0 + cdec*cdec0*cdra) - 1.
+
+infinite_vals=where(NOT float(finite(n_tracked)),n_count)
```

Comment:
Would be better written as `infinite_vals=where(finite(n_tracked, /inf),n_count)`

Reply:
done